tests/test_license_files.py

```diff
@@ -0,0 +1,52 @@
+"""
+Skeleton tests for `license_files` from `setup.cfg`.  This is a `setuptools`
+extension which is not yet ratified by the Python community.
+
+`license_files` is the subject of a PEP: https://peps.python.org/pep-0639/
```

Comment:
I think this should be restructured text markup, although we don't actually generate documentation for these files (yet?) so it's not a blocker

Reply:
Yep, I'll fix that up now… just got to look up what the syntax is for RST. (I'm too used to Markdown, sadly the Markdown situation with Sphinx is sketchy so RST really is the better choice)

Reply:
Haha yeah I can relate